UPGRADING

```diff
@@ -532,6 +532,8 @@ PHP 8.0 UPGRADE NOTES
   . parse_str() can no longer be used without specifying a result array.
   . fgetss() has been removed.
   . The string.strip_tags filter has been removed.
+  . strspn() and strcspn() now return false and throw a ValueError when the
+    start or length argument exceed the bounds of the string.
```

Comment:
"now return false and throw a ValueError" doesn't make sense together -- if it throws a ValueError, it can't return anything. I would just drop the mention of false here.

Reply:
Uh, indeed. I misunderstood [the change in this line](https://github.com/php/php-src/commit/5d9ab53a5d53f11a18ae11ed31b17ff87c8d52a7#diff-96c05a0551ccbeca3ff610c111f88b54a0d727888cf353f477ea8dc951a18637L545).